### ISSUE TITLE
Add something to Moodle Help pages for grid format

### ISSUE THREAD

**Issue by thepurpleblob** [gjb2048/moodle-format_grid#101]
Currently clicking the 'Help with this page' link on a course page using Grid format leads to...

https://docs.moodle.org/37/en/course/view/grid  (or equivalent)

which is an unpopulated page. 

Some useful content would be good.

**Comment by gjb2048** [comment]
Hi Howard,

Done.

I'm getting worried about you though.  Recently your tone has been coming across as a bit short, stressed, sarcastic and under pressure.  Are you ok?

G

**Comment by thepurpleblob** [comment]
Thanks... much appreciated. I did wonder if taking you to the course format page is the correct action for that link. However, everybody + dog loves the grid format around here so we kept getting this pointed out (I'm surprised anybody even found the help link).

And, yes, I'm a miserable, grumpy b*****d. It's taken *years* to cultivate. Don't take it personally. I'm delightful in person. Sometimes ;-)
